python/py-telethon/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-telethon
+version             1.24.0
+revision            0
+license             MIT
+maintainers         nomaintainer
+
+description         Full-featured Telegram client library.
+long_description    Telethon is an asyncio Python 3 MTProto library to \
+                    interact with Telegram's API as a user or through \
+                    a bot account (bot API alternative).
+
+homepage            https://github.com/LonamiWebs/Telethon
+
+python.versions     37 38 39 310
+
+distname            [string totitle ${distname}]
+
+checksums           rmd160  7ebef3569ba02fcdb5886efba4b5d8183ad6b4a7 \
+                    sha256  818cb61281ed3f75ba4da9b68cb69486bed9474d2db4e0aa16e482053117452c \
+                    size    464470
+
+if {${name} ne ${subport}} {
+    depends_lib-append  \
```

Comment:
this should likely be `depends_build-append`?

Reply:
Thanks done.